app/Repositories/Project/ProjectRepository.php

```diff
@@ -95,6 +96,11 @@ public function clone($authUser, Project $project, $token)
                 'cloned_from' => $project->id,
             ];
 
+            if($organization_id) {
+                $data['organization_id'] = $organization_id;
+                $data['organization_visibility_type_id'] = 1;
+            }
+
```

Comment:
Formatting issue; need a space after 'if'

Reply:
Added space after "if" in ProjectRepository@clone() and ProjectRepository@populateOrderNumber().